From 0969474ed5891c1d953512df8b8635fe0003da74 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Tue, 21 Jul 2026 03:28:58 +0200
Subject: [PATCH 123/125] rpcd: update to Git HEAD (2026-07-19)

e37ed9d81469 file: re-authorize ACL against resolved path to close symlink bypass

Link: https://github.com/openwrt/openwrt/pull/24329
(cherry picked from commit b5191da03cf4864416afee8ef5542e1ffe596d41)
Link: https://github.com/openwrt/openwrt/pull/24351
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
(cherry picked from commit 029fec82456af6d7bd17c8aaff1e6bee1f477505)
---
 package/system/rpcd/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/system/rpcd/Makefile b/package/system/rpcd/Makefile
index b0b3128838..a08ac888d0 100644
--- a/package/system/rpcd/Makefile
+++ b/package/system/rpcd/Makefile
@@ -12,9 +12,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/rpcd.git
-PKG_MIRROR_HASH:=b15b42e9f27a35030f488d7a61a3b2f505f79b5675455b20f1336acb1d99686a
-PKG_SOURCE_DATE:=2026-06-04
-PKG_SOURCE_VERSION:=28faf6403792d25b9826043aaf37880624c19568
+PKG_MIRROR_HASH:=ca2498a25c69e6b6e8fe1bcc5865a244e409674b1ac2e2e36c0e4bb04b11aba8
+PKG_SOURCE_DATE:=2026-07-19
+PKG_SOURCE_VERSION:=e37ed9d814699098eb7e26c8b33c054840782dfb
 PKG_MAINTAINER:=Jo-Philipp Wich <jo@mein.io>
 
 PKG_LICENSE:=ISC
-- 
2.34.1

